modules/openapi-generator/src/main/resources/Java/apiException.mustache

```diff
@@ -37,7 +37,7 @@ public class ApiException extends{{#useRuntimeException}} RuntimeException {{/us
     }
 
     public ApiException(int code, Map<String, List<String>> responseHeaders, String responseBody) {
-        this((String) null, (Throwable) null, code, responseHeaders, responseBody);
+        this(code + " " + responseBody, (Throwable) null, code, responseHeaders, responseBody);
```

Comment:
@brendandburns thanks for the PR. What about using the following instead to clearly indicate the code and response body?
```
        this(String.format("Code: %i. Response body: %s", code, responseBody), (Throwable) null, code, responseHeaders, responseBody);
```

Reply:
Done.